docs/source/getting_started/dataset_creation/dataset_creation_azure_storage.rst

```diff
@@ -32,8 +32,8 @@ Preparing your data
 For the :ref:`lightly-command-line-tool` to be able to create embeddings and extract metadata from your data, `lightly-magic` needs to be able to access your data. You can download/sync your data from Azure blob storage.
  
 1. Install AzCopy cli by following the `guide of Azure <https://docs.microsoft.com/en-us/azure/storage/common/storage-use-azcopy-v10>`_
-2. To sync your data from the container to your local machine (or vice versa), you need a shared access token. On your *storage account* page of the Azure portal, on the left, go to "Security + networking > Shared access signature". Generate a shared access signature (SAS) which allows access to the container and objects.
-3. Copy the SAS token and use the following command to sync the Azure blob storage with your local folder:
+2. To sync your data from the container to your local machine (or vice versa), you need a shared access token. On your `storage account`` page of the Azure portal, on the left, go to **"Security + networking"** > **"Shared access signature"**. Generate a shared access signature (SAS) which allows access to the container and objects.
+3. Copy the `SAS token`` and use the following command to sync the Azure blob storage with your local folder:
```

Comment:
There seem to be some single ` and some double ``. I guess all of them should be single?